lib/success.js

```diff
@@ -275,3 +290,25 @@ export function buildAssociatedPRsQuery(shas) {
     }
   `;
 }
+
+/**
+ * Build associatedPRs (to issue-like object with `pull_request` property) from the GraphQL repository response
+ * @param {object} repository
+ * @returns {object[]}
+ */
+function buildAssociatedPRs(repository) {
+  const associatedPRs = [];
+  for (const commit in repository) {
+    for (const node of repository[commit].associatedPullRequests.nodes) {
+      const pr = {
+        number: node.number,
+        body: node.body,
+        pull_request: {
+          url: node.url,
+        },
+      };
+      associatedPRs.push(pr);
+    }
+  }
+  return associatedPRs;
+}
```

Comment:
- Utility to help build the PR object as `issue` value required for filtering in the `success|failCommentCondition`